### DISCUSSION TITLE
New Tesla Replacing Old Tesla - can I name it the same?

### DISCUSSION THREAD

**Discussion by steve28** [teslamate-org/teslamate#2514] [General]
I just got a new 2022 M3 to replace my old 2018 M3... I would like to keep the data from the two separate, but I want to name my car the same... What will occur if I name the new car the same as the old car?

**Comment by natrlhy** [comment]
They will remain separate as they are not the same car/VIN. Internally in TeslaMate will be car_id=1 and the other will be car_id=2. As long as they are tied to the same tesla account, too

**Reply by steve28** [reply]
Hmm... I named the new car the same as the old car ... Is there a way I can change the name of the old car?   It doesn't seem to be recording the drives I drove to work with No Name then I changed it when I got in the car to drive home... I see the car the status page with current charge and all that, but I do not see anything in graphana for the new one?

**Reply by cwanja** [reply]
Could rename it via the database. Unless you still have the old car via your Tesla account. 

**Remember** to always backup your database first.

**Reply by steve28** [reply]
I know nothing about how to muck with the database.... I don't supposed it's a simple command?

I looked at [this](https://github.com/adriankumpf/teslamate/discussions/1938#discussioncomment-1252831), but this seems different than what I need.  I need to set the name of car_id=1 to a new value and leave car_id=2 untouched

**Reply by cwanja** [reply]
```
docker-compose exec -T database psql -U teslamate << .
select id, name, vin from cars;
.
```
This will list the cars database ID, Tesla friendly name and VIN. Determine the ID you want to rename.

```
docker-compose exec -T database psql -U teslamate << .                            
    UPDATE cars SET name='new_car_name' WHERE id='database-id-for-old-Model3';
.
```
As you mentioned, it is _probably_ going to be ID 1. But the first command will help verify that. As always, remember to backup. https://docs.teslamate.org/docs/maintenance/backup_restore#backup

**Reply by steve28** [reply]
Worked great! Thanks!

**Comment by cwanja** [comment]
Probably could write a one liner for it. Will look at it tomorrow.

**Comment by MrNickIE** [comment]
Thanks for this!  I hit same issue today!
